shell/platform/embedder/embedder.cc

```diff
@@ -115,34 +141,32 @@ static bool IsRendererValid(const FlutterRendererConfig* config) {
 
   switch (config->type) {
     case kOpenGL:
+#if SHELL_ENABLE_GL
```

Comment:
Same comment as above. There is no need to have a different embedder dylibs for OpenGL and Vulkan. Once will suffice since proc address resolution occurs at runtime. This should also make these ifdef guards unnecessary.